code/datums/traits/negative.dm

```diff
@@ -274,3 +274,20 @@
 		dumb_thing = FALSE //only once per life
 		if(prob(1))
 			new/obj/item/reagent_containers/food/snacks/pastatomato(get_turf(H)) //now that's what I call spaghetti code
+
+/datum/quirk/phobia
+	name = "Phobia"
+	desc = "You've had a traumatic past, that has scared you for life while dealing with your greatest fear."
+	value = -3 // It can hardstun you. You can be a job that your phobia targets...
```

Comment:
This value is pretty high for something that's only shit if you have bad RNG.

Reply:
Fair lower to 2 seems fire then